governator-legacy/src/main/java/com/netflix/governator/guice/InternalAutoBindModuleBootstrapModule.java

```diff
@@ -63,6 +63,7 @@ private Module bindAutoBindSingletons(BootstrapBinder binder) {
                         "@AutoBindSingleton(multiple=true) value cannot be set for Modules");
 
                 LOG.info("Found @AutoBindSingleton annotated module : {} ", clazz.getName());
+                LOG.info("***** @AutoBindSingleton for module {} is deprecated as of 10/10/2015.  Modules should be added directly to the injector or via install({}.class). See https://github.com/Netflix/governator/wiki/Auto-Binding", clazz.getName(), clazz.getSimpleName());
```

Comment:
For the warning, the application owner that needs to explicitly install the module may be different than the library owner that needs to remove the AutoBindSingleton annotation. It seems like the primary consumer for the warning would be the application owner and as long as they are doing the right thing they probably shouldn't see the notice.